package.json

```diff
@@ -13,6 +13,8 @@
     "@types/react-bootstrap": "^0.32.14",
     "@types/react-lottie": "^1.2.0",
     "@types/react-router-dom": "^4.3.1",
+    "@types/react-stripe-elements": "^1.1.10",
+    "axios": "^0.18.0",
```

Comment:
Are we using this package? I'm not seeing it imported in any of the files. If so lets remove it.